modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-lvmcluster_{context}"]
+= About the LVMCluster custom resource
+
+After you have installed {lvms}, you must create an `LVMCluster` custom resource (CR) on a worked node.
+
+You can configure the `LVMCluster` CR to perform the following actions:
+
+* Create LVM volume groups.
+* Configure a list of devices that you want to add to an LVM volume group. 
+* Configure the requirements to select the nodes on which you want to create an LVM volume group (VG), and the thin pool configuration for the VG.
+* Force wipe the selected devices.
+
+You can create an `LVMCluster` CR using the OpenShift CLI (`oc`) or the {product-title} web console.
+
+include::snippets/lvms-creating-lvmcluster.adoc[]
+
+The `LVMCluster` CR contains the following fields:
+
+`deviceClasses`:: Contains the configuration to assign the local storage devices to the LVM volume groups. 
++
+To create multiple device storage classes, create an array for each storage class. 
++
+If you add or remove a `deviceClass`, then the update reflects in the `LVMCluster` CR only after deleting and recreating the `TopoLVM-Node` pod. 
```

Comment:
```suggestion
If you add or remove a `deviceClass` after already having setup deviceClasses previously, then the update reflects in the `LVMCluster` CR only after deleting and recreating the `TopoLVM-Node` pod. 
```

Reply:
Updated it as follows:
"After creating an `LVMCluster` CR, if you add or remove a device class, the update reflects in the `LVMCluster` CR only after deleting and recreating the `TopoLVM-Node` pod."